Reviewed general punctuation and fixed some broken links

sergiopaniego opened the pull request:
In this PR I've: 
* Reviewed the general punctuation of the units and updated it.
* Fixed some links that appeared broken. 

I've used https://moon-ci-docs.huggingface.co/docs/computer-vision-course/pr_246/en/Unit%200%20-%20Welcome/welcome as a source to find the issues.

Let me know if the course is updated elsewhere and I'll update it accordingly. 
This PR may cause merge issues with https://github.com/johko/computer-vision-course/pull/250, so I could update it once the former is merged

merveenoyan reviewed (commented):
We do not have dots on headings, resources, clauses (that aren't sentences), end of parenthesis that already have dots 
Also if you have time and if you feel like it, you can fix broke image links (I think there's a few in chapter 9)

sergiopaniego commented:
Thanks for the feedback @merveenoyan! Would you like me to make modifications regarding the punctuation?
I'll review Chapter 9 for broken image links. 
For now, I've merged the updated main branch :smile:

sergiopaniego commented on chapters/en/Unit 9 - Model Optimization/intro_to_model_optimization.mdx:
```diff
@@ -16,12 +16,12 @@ As we already know, optimizing the model is important in before the deployment s
 1. Pruning: Pruning is the process of eliminating redundant or unimportant connections in the model. This aims to reduce model size and complexity.
 
-![Pruning](https://huggingface.co/datasets/hf-vision/course-assets/raw/main/pruning.png)
+![Pruning](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/pruning.png)
```
Can we consider the links in Unit 9 fixed in this way?

sergiopaniego commented on chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/nerf.mdx:
```diff
@@ -144,7 +144,7 @@ visualize_grid(grid, encoded_grid, resolution)
 The output should look something like the image below:
 
-![encoding](https://huggingface.co/datasets/hf-vision/course-assets/blob/main/nerf_encodings.png)
+![encoding](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/nerf_encodings.png)
```
This Unit image link is also updated

sergiopaniego commented on chapters/en/Unit 2 - Convolutional Neural Networks/introduction.mdx:
```diff
@@ -70,7 +70,7 @@ The feature map will be the same size as the original data. The result of the co
 
 <div class="flex justify-center">
-    <img src="https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/CNNs/2d-feature_map.png" alt="2D Feature Map">
+    <img src="https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/CNNs/2d_feature_map.png" alt="2D Feature Map">
```
Image broken link updated here

sergiopaniego commented:
I've indicated where the update links are located. 
I could even generate a separate PR just with the broken links if that's easier for merging :smiley:

merveenoyan commented on chapters/en/unit5/generative-models/gans-vaes/stylegan.mdx:
```diff
@@ -22,15 +22,15 @@ TL DR; StyleGAN is a special modification made to the architectural style of the
-2. AdaIN (Adaptive Instance Normalisation)
-3. Concatenation of Noise vector
+Diagram (b) is the proposed StyleGAN architecture. It has the following main components:
+1. A mapping network.
```
these aren't sentences so no need

merveenoyan commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -114,27 +114,27 @@ Let's focus on a single point - the top left corner of the laptop. As per equati
 ![Rectified Left Image](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/3d_stereo_vision_images/rectified_left_frame.jpg?download=true)
 
-Rectified Right Image 
+Rectified Right Image. 
```
these aren't sentences so no need

merveenoyan commented on chapters/en/unit9/tools_and_frameworks.mdx:
```diff
@@ -88,22 +88,22 @@ TensorRT is available as a pip package, `tensorrt`. To install the package, run
 - Write an application once, deploy it anywhere on your preferred device, language and OS.
-- has minimal external dependencies 
-- Reduces first-inference latency by using the CPU for initial inference and then switching to another device once the model has been compiled and loaded to memory
+- has minimal external dependencies.
```
these are subsentences in form of lists so no need

merveenoyan commented on chapters/en/unit9/tools_and_frameworks.mdx:
```diff
@@ -142,24 +142,24 @@ Optimum is available as a pip package, `optimum`. To install the package, run th
 The benefits of using EdgeTPU includes:
-- Complements Cloud TPU and Google Cloud services to provide an end-to-end, cloud-to-edge, hardware + software infrastructure for AI-based solutions deployment
-- High performance in a small physical and power footprint
+- Complements Cloud TPU and Google Cloud services to provide an end-to-end, cloud-to-edge, hardware + software infrastructure for AI-based solutions deployment.
```
these are subsentences

merveenoyan reviewed (commented):
Thanks a lot! helps greatly :)

sergiopaniego commented:
I've updated the files with all the suggestions, accepted your commits, and restored a broken link to a chapter.
Please let me know if there's anything else that needs attention!

> @sergiopaniego given the amount of fixes I think you should add your name to credits

How should I add myself @merveenoyan?

Thanks for the in-depth review!

merveenoyan commented on chapters/en/unit0/welcome/welcome.mdx:
```diff
@@ -12,29 +12,30 @@ On this page, you can find how to join the learners community, make a submission
 
-1. Training/fine-tuning a Model
-2. Building an application and hosting it on Hugging Face Spaces
+1. Training/fine-tuning a Model.
```
```suggestion
1. Training/fine-tuning a model
```

merveenoyan commented on chapters/en/unit0/welcome/welcome.mdx:
```diff
@@ -12,29 +12,30 @@ On this page, you can find how to join the learners community, make a submission
 
-- [Getting started with Gradio](https://huggingface.co/learn/nlp-course/chapter9/1?fw=pt#introduction-to-gradio)
-- [How to share your application on 🤗 Spaces](https://huggingface.co/learn/nlp-course/chapter9/4?fw=pt)
+- [Getting started with Gradio](https://huggingface.co/learn/nlp-course/chapter9/1?fw=pt#introduction-to-gradio).
```
```suggestion
- [Getting started with Gradio](https://huggingface.co/learn/nlp-course/chapter9/1?fw=pt#introduction-to-gradio)
```

merveenoyan commented on chapters/en/unit0/welcome/welcome.mdx:
```diff
@@ -12,29 +12,30 @@ On this page, you can find how to join the learners community, make a submission
-- [Getting started with Gradio](https://huggingface.co/learn/nlp-course/chapter9/1?fw=pt#introduction-to-gradio)
-- [How to share your application on 🤗 Spaces](https://huggingface.co/learn/nlp-course/chapter9/4?fw=pt)
+- [Getting started with Gradio](https://huggingface.co/learn/nlp-course/chapter9/1?fw=pt#introduction-to-gradio).
+- [How to share your application on 🤗 Spaces](https://huggingface.co/learn/nlp-course/chapter9/4?fw=pt).
```
```suggestion
- [How to share your application on 🤗 Spaces](https://huggingface.co/learn/nlp-course/chapter9/4?fw=pt)
```

merveenoyan commented on chapters/en/unit0/welcome/welcome.mdx:
```diff
@@ -52,7 +53,7 @@ As a computer vision course learner, you may find the following set of channels
 - `#computer-vision`: a catch-all channel for everything related to computer vision.
 - `#cv-study-group`: a place to exchange ideas, ask questions about specific posts and start discussions.
-- `#3d`: a channel to discuss aspects of computer vision specific to 3D computer vision
+- `#3d`: a channel to discuss aspects of computer vision specific to 3D computer vision.
```
```suggestion
- `#3d`: a channel to discuss aspects of computer vision specific to 3D computer vision
```

merveenoyan commented on chapters/en/unit0/welcome/welcome.mdx:
```diff
@@ -52,7 +53,7 @@ As a computer vision course learner, you may find the following set of channels
 
 - `#computer-vision`: a catch-all channel for everything related to computer vision.
 - `#cv-study-group`: a place to exchange ideas, ask questions about specific posts and start discussions.
```
```suggestion
- `#cv-study-group`: a place to exchange ideas, ask questions about specific posts and start discussions
```

merveenoyan commented on chapters/en/unit0/welcome/welcome.mdx:
```diff
@@ -52,7 +53,7 @@ As a computer vision course learner, you may find the following set of channels
 
 - `#computer-vision`: a catch-all channel for everything related to computer vision.
```
```suggestion
- `#computer-vision`: a catch-all channel for everything related to computer vision
```

merveenoyan commented on chapters/en/unit10/blenderProc.mdx:
```diff
@@ -103,18 +103,22 @@ You can install BlenderProc via pip:
-![colors](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/colors.png)
-![normals](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/normals.png)
-![depth](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/depth.png)
+![colors](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/colors.png).
```
```suggestion
![colors](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/colors.png)
```

merveenoyan commented on chapters/en/unit10/blenderProc.mdx:
```diff
@@ -103,18 +103,22 @@ You can install BlenderProc via pip:
-![normals](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/normals.png)
-![depth](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/depth.png)
+![colors](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/colors.png).
+![normals](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/normals.png).
```
```suggestion
![normals](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/normals.png)
```

merveenoyan commented on chapters/en/unit10/blenderProc.mdx:
```diff
@@ -103,18 +103,22 @@ You can install BlenderProc via pip:
-![depth](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/depth.png)
+![colors](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/colors.png).
+![normals](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/normals.png).
+![depth](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/depth.png).
```
```suggestion
![depth](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/synthetic-data-creation-PBR/depth.png)
```

merveenoyan commented on chapters/en/unit10/point_clouds.mdx:
````diff
@@ -22,19 +22,19 @@ The 3D Point Data is mainly used in self-driving capabilities, but now other AI
+We will be using the python library [point-cloud-utils](https://github.com/fwilliams/point-cloud-utils), and [open-3d](https://github.com/isl-org/Open3D), which can be installed by:
 
 ```bash
  pip install point-cloud-utils
````
```suggestion
pip install point-cloud-utils
```

merveenoyan commented on chapters/en/unit10/synthetic_datasets.mdx:
```diff
@@ -39,8 +39,8 @@ Semantic segmentation is vital for autonomous vehicles to interpret and navigate
 |---------------------|--------------|-------------|----------------|---------------------|---------------------|
-| Virtual KITTI 2      | 2020          | Virtual Worlds as Proxy for Multi-Object Tracking Analysis | [Virtual KITTI 2](https://arxiv.org/pdf/2001.10773.pdf) |  | [Website](https://europe.naverlabs.com/Research/Computer-Vision/Proxy-Virtual-Worlds/) |
-| ApolloScape | 2019 | Compared with existing public datasets from real scenes, e.g. KITTI [2] or Cityscapes [3], ApolloScape contains much large and richer labeling including holistic semantic dense point cloud for each site, stereo, per-pixel semantic labeling, lane-mark labeling, instance segmentation, 3D car instance, high accurate location for every frame in various driving videos from multi ...
+| Virtual KITTI 2      | 2020          | Virtual Worlds as Proxy for Multi-Object Tracking Analysis. | [Virtual KITTI 2](https://arxiv.org/pdf/2001.10773.pdf) |  | [Website](https://europe.naverlabs.com/Research/Computer-Vision/Proxy-Virtual-Worlds/) |
```
```suggestion
| Virtual KITTI 2      | 2020          | Virtual Worlds as Proxy for Multi-Object Tracking Analysis | [Virtual KITTI 2](https://arxiv.org/pdf/2001.10773.pdf) |  | [Website](https://europe.naverlabs.com/Research/Computer-Vision/Proxy-Virtual-Worlds/) |
```

merveenoyan commented on chapters/en/unit2/cnns/convnext.mdx:
```diff
@@ -9,16 +9,18 @@ ConvNext represents a significant improvement to pure convolution models by inco
-- Inverted Bottleneck
-- Large Kernel Sizes
-- Micro Design
+- Training Techniques.
```
```suggestion
- Training techniques
```

merveenoyan commented on chapters/en/unit2/cnns/convnext.mdx:
```diff
@@ -9,16 +9,18 @@ ConvNext represents a significant improvement to pure convolution models by inco
-- Large Kernel Sizes
-- Micro Design
+- Training Techniques.
+- Macro Design.
```
```suggestion
- Macro design
```

merveenoyan commented on chapters/en/unit2/cnns/convnext.mdx:
```diff
@@ -9,16 +9,18 @@ ConvNext represents a significant improvement to pure convolution models by inco
-- Micro Design
+- Training Techniques.
+- Macro Design.
+- ResNeXt-ify.
```
```suggestion
- ResNeXt-ify
```

merveenoyan commented on chapters/en/unit2/cnns/convnext.mdx:
```diff
@@ -9,16 +9,18 @@ ConvNext represents a significant improvement to pure convolution models by inco
+- Training Techniques.
+- Macro Design.
+- ResNeXt-ify.
+- Inverted Bottleneck.
```
```suggestion
- Inverted bottleneck
```

merveenoyan commented on chapters/en/unit2/cnns/convnext.mdx:
```diff
@@ -9,16 +9,18 @@ ConvNext represents a significant improvement to pure convolution models by inco
+- Macro Design.
+- ResNeXt-ify.
+- Inverted Bottleneck.
+- Large Kernel Sizes.
```
```suggestion
- Large kernel sizes
```

merveenoyan commented on chapters/en/unit2/cnns/convnext.mdx:
```diff
@@ -9,16 +9,18 @@ ConvNext represents a significant improvement to pure convolution models by inco
+- ResNeXt-ify.
+- Inverted Bottleneck.
+- Large Kernel Sizes.
+- Micro Design.
```
```suggestion
- Micro design
```

merveenoyan commented on chapters/en/unit3/vision-transformers/detr.mdx:
````diff
@@ -138,12 +138,12 @@ class DETR(nn.Module):
 h = self.conv(x)
 ```
-they are declared in the `__init__` function
+they are declared in the `__init__` function.
````
```suggestion
They are declared in the `__init__` function.
```

merveenoyan commented on chapters/en/unit3/vision-transformers/vision-transformer-for-objection-detection.mdx:
```diff
@@ -15,17 +15,17 @@ This section will describe how object detection tasks are achieved using Vision
 There are a lot of of applications around object detection. One of the most significant examples is in the field of autonomous driving, where object detection is used to detect different objects (like pedestrians, road signs, traffic lights, etc)  around the car that become one of the inputs for taking decisions.  
 
-To deepen your understanding of the ins-and-outs of object detection, check out our [dedicated chapter](/chapters/en/Unit%206%20-%20Basic%20CV%20Tasks/object_detection.mdx) on Object Detection 🤗
+To deepen your understanding of the ins-and-outs of object detection, check out our [dedicated chapter](/learn/computer-vision-course/unit6/basic-cv-tasks/object_detection) on Object Detection 🤗.
```
```suggestion
To deepen your understanding of the ins-and-outs of object detection, check out our [dedicated chapter](https://huggingface.co/learn/computer-vision-course/unit6/basic-cv-tasks/object_detection) on Object Detection 🤗.
```
just in case

merveenoyan commented on chapters/en/unit4/multimodal-models/clip-and-relatives/clip.mdx:
```diff
@@ -55,8 +55,8 @@ probs = logits_per_image.softmax(dim=1)
 After executing this code, we got the following probabilities:
-- "a photo of a cat": 99.49%
-- "a photo of a dog": 0.51% 
+- "a photo of a cat": 99.49%.
```
```suggestion
- "a photo of a cat": 99.49%
```

merveenoyan commented on chapters/en/unit4/multimodal-models/clip-and-relatives/clip.mdx:
```diff
@@ -55,8 +55,8 @@ probs = logits_per_image.softmax(dim=1)
-- "a photo of a cat": 99.49%
-- "a photo of a dog": 0.51% 
+- "a photo of a cat": 99.49%.
+- "a photo of a dog": 0.51%.
```
```suggestion
- "a photo of a dog": 0.51%
```

merveenoyan commented on chapters/en/unit5/generative-models/diffusion-models/introduction.mdx:
```diff
@@ -2,17 +2,17 @@
-- Major sub categories of Diffusion models
-- Use cases of Diffusion models
-- Drawback in Diffusion models
+- What are diffusion models and how do they differ from GANs.
```
```suggestion
- What are diffusion models and how do they differ from GANs
```

merveenoyan commented on chapters/en/unit5/generative-models/diffusion-models/introduction.mdx:
```diff
@@ -2,17 +2,17 @@
-- Use cases of Diffusion models
-- Drawback in Diffusion models
+- What are diffusion models and how do they differ from GANs.
+- Major sub categories of Diffusion models.
```
```suggestion
- Major sub categories of diffusion models
```

merveenoyan commented on chapters/en/unit5/generative-models/diffusion-models/introduction.mdx:
```diff
@@ -2,17 +2,17 @@
-- Drawback in Diffusion models
+- What are diffusion models and how do they differ from GANs.
+- Major sub categories of Diffusion models.
+- Use cases of Diffusion models.
```
```suggestion
- Use cases of diffusion models
```

merveenoyan commented on chapters/en/unit5/generative-models/diffusion-models/introduction.mdx:
```diff
@@ -2,17 +2,17 @@
+- What are diffusion models and how do they differ from GANs.
+- Major sub categories of Diffusion models.
+- Use cases of Diffusion models.
+- Drawback in Diffusion models.
```
```suggestion
- Drawback in diffusion models
```

merveenoyan commented on chapters/en/unit5/generative-models/diffusion-models/stable-diffusion.mdx:
```diff
@@ -3,8 +3,8 @@ This chapter introduces the building blocks of Stable Diffusion which is a gener
 What will you learn from this chapter?
-- Fundamental components of Stable Diffusion
-- How to use `text-to-image`, `image2image`, inpainting pipelines 
+- Fundamental components of Stable Diffusion.
```
```suggestion
- Fundamental components of Stable Diffusion
```

merveenoyan commented on chapters/en/unit5/generative-models/introduction/introduction.mdx:
```diff
@@ -15,9 +15,9 @@ these tasks can be expanded into more complex processes such as semantic segment
-* noise to image (DCGAN)
-* text to image (diffusion models)
-* image to image (StyleGAN, cycleGAN, diffusion models)
+* noise to image (DCGAN).
```
```suggestion
* noise to image (DCGAN)
```

merveenoyan commented on chapters/en/unit5/generative-models/practical-applications/ethical-issues.mdx:
```diff
@@ -4,9 +4,9 @@ The widespread adoption of AI-powered image editing tools raises significant con
-- Impact of such AI images/videos on society
-- Current approaches to tackle the issues
-- Future scope
+- Impact of such AI images/videos on society.
```
```suggestion
- Impact of such AI images/videos on society
```

merveenoyan commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -15,8 +15,8 @@ Figure 1: Image formation using single camera
-1. Single image of a scene point P
-2. Pixel coordinates of point P in the image 
+1. Single image of a scene point P.
+2. Pixel coordinates of point P in the image.
```
```suggestion
2. Pixel coordinates of point P in the image
```

merveenoyan commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -44,18 +44,18 @@ Since there are many different positions and orientations for the camera locatio
 ![Figure 2: Image formation using 2 cameras](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/3d_stereo_vision_images/image_formation_simple_stereo.jpg?download=true)
 
-Figure 2: Image formation using 2 cameras 
+Figure 2: Image formation using 2 cameras. 
```
```suggestion
Figure 2: Image formation using 2 cameras
```

merveenoyan commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -68,12 +68,12 @@ With the above configuration in place, we have the below equations which map a p
-*  x,y,z are 3D coordinates of the point P (any unit like cm, feet, etc can be used)    
-*  \\(O\_x\\)  and  \\(O\_y\\)  refer to pixel coordinates of the principal point     
-*  b is called the baseline and refers to the distance between the left and right cameras. Same units are used for both b and x,y,z coordinates (any unit like cm, feet, etc can be used)    
+*  x,y,z are 3D coordinates of the point P (any unit like cm, feet, etc can be used).
```
```suggestion
*  x, y, z are 3D coordinates of the point P (any unit like cm, feet, etc can be used).
```

merveenoyan commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -97,10 +97,10 @@ We'll work through an example, capture some images, and perform some calculation
 The left and right cameras in OAK-D Lite are oriented similarly to the geometry of the simplified solution detailed above. The baseline distance between the left and right cameras is 7.5cm. Left and right images of a scene captured using this device are shown below. The figure also shows these images stacked horizontally with a red line drawn at a constant height (i.e. at a constant v value ). We ...
 
-Raw Left Image
+Raw Left Image.
```
```suggestion
Raw Left Image
```

merveenoyan commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -97,10 +97,10 @@ We'll work through an example, capture some images, and perform some calculation
 ![Raw Left Image](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/3d_stereo_vision_images/unrectified_left_frame.jpg?download=true)
 
-Raw Right Image 
+Raw Right Image. 
```
```suggestion
Raw Right Image
```

merveenoyan commented on chapters/en/unit8/introduction/brief_history.mdx:
```diff
@@ -2,17 +2,17 @@
 ## 1853: Anaglyph 3D
 
-- **Pioneer**: Louis Ducos du Hauron
+- **Pioneer**: Louis Ducos du Hauron.
```
```suggestion
- **Pioneer**: Louis Ducos du Hauron
```

merveenoyan commented on chapters/en/unit8/introduction/brief_history.mdx:
```diff
@@ -22,7 +22,7 @@
 ## 1979: Autostereograms (Magic Eye Images)
 
-- **Creator**: Christopher Tyler
+- **Creator**: Christopher Tyler.
```
```suggestion
- **Creator**: Christopher Tyler
```

merveenoyan reviewed (approved):
Thanks! be mindful that we are looking for atomic PRs otherwise it will take a long time to merge changes

merveenoyan commented:
@sergiopaniego when we make comments we would like to make them general, please reflect them across the text or I have to fix them one by one which takes a lot of time

merveenoyan commented:
@sergiopaniego you can give yourself credit on credits section in 0th chapter

sergiopaniego commented on chapters/en/unit8/3d_measurements_stereo_vision.mdx:
```diff
@@ -97,10 +97,10 @@ We'll work through an example, capture some images, and perform some calculation
 The left and right cameras in OAK-D Lite are oriented similarly to the geometry of the simplified solution detailed above. The baseline distance between the left and right cameras is 7.5cm. Left and right images of a scene captured using this device are shown below. The figure also shows these images stacked horizontally with a red line drawn at a constant height (i.e. at a constant v value ). We ...
 
-Raw Left Image
+Raw Left Image.
```
I'll do it! Thanks for all the suggestions and the review :smile:

sergiopaniego commented:
> @sergiopaniego when we make comments we would like to make them general, please reflect them across the text or I have to fix them one by one which takes a lot of time

Thank you for all the suggestions and your thorough review. Apologies for my delayed responses (been busy with work and a conference).

I've accepted all your suggestions. I'll conduct a detailed review of the entire PR to ensure that all the suggestions are reflected.

sergiopaniego commented:
Hi @merveenoyan :smile: I've conducted the review and the PR is now ready to be checked. 

I've removed some punctuation thanks to your suggestions and updated some broken links that I've found while reviewing (https://github.com/johko/computer-vision-course/commit/1c20a24300d48296ee7f3a0c2ba2124593179cc2)

johko reviewed (approved):
a very late review, but LGTM :+1: 

will merge it (finally)

sergiopaniego commented:
Thanks @johko! :smile:
